diff --git PKGBUILD PKGBUILD
index 3c58f02..47acd1f 100644
--- PKGBUILD
+++ PKGBUILD
@@ -16,7 +16,7 @@ url='https://www.qt.io'
 license=(GPL2 LGPL)
 pkgdesc='Generates bindings for C++ libraries using CPython source code (git version)'
 makedepends=(clang llvm cmake git libxslt qt5-xmlpatterns python-sphinx)
-source=("$pkgbase::git+https://code.qt.io/pyside/pyside-setup.git")
+source=("$pkgbase::git+https://code.qt.io/pyside/pyside-setup.git#branch=5.14")
 sha256sums=('SKIP')
 
 pkgver() {
@@ -29,7 +29,6 @@ prepare() {
   cd "${srcdir}/${pkgbase}"
 
   mkdir -p build
-  sed -e 's|0307FFFF|0308FFFF|' -i sources/shiboken2/libshiboken/pep384impl.h # Support python 3.8
 }
 
 build() {
