--- /usr/lib/python3.8/site-packages/shiboken2/__init__.py.orig	2020-02-07 18:22:35.178361555 +0000
+++ /usr/lib/python3.8/site-packages/shiboken2/__init__.py	2020-02-07 18:22:56.476144293 +0000
@@ -1,30 +1,2 @@
 __version__ = "5.14.0"
 __version_info__ = (5, 14, 0, "a", "1")
-
-# PYSIDE-932: Python 2 cannot import 'zipfile' for embedding while being imported, itself.
-# We simply pre-load all imports for the signature extension.
-# Also, PyInstaller seems not always to be reliable in finding modules.
-# We explicitly import everything that is needed:
-import sys
-import os
-import zipfile
-import base64
-import marshal
-import io
-import contextlib
-import textwrap
-import traceback
-import types
-import struct
-import re
-import tempfile
-import keyword
-import functools
-if sys.version_info[0] == 3:
-    # PyInstaller seems to sometimes fail:
-    import typing
-
-from .shiboken2 import *
-
-# Trigger signature initialization.
-type.__signature__
